Nonprofit-Exchange-Hub/web-app
# remove use of mockData (assets) and replace with BE data, other small fixes

## Dev Summary


frontend has been using hardcoded data. this uses data from the db and adds seed data




## Test Plan


repro steps:

see BE data
1. from server, before starting, seed new assets `npm run seed:drop`
2. start servers
3. localhost:3000
4. open dev tools network tab
see that we made 2 calls to /assets and that data is what we're seeing as asset cards
5. click on one of the asset cards
6. see asset show page that matches card

see FE routing is fixed
1. localhost:3000
2. click `make a post` button
3. click `goods`

## Resources

- [orm find function](https://typeorm.io/find-options)
